src/main/kotlin/com/craftschemers/hub/HubPlayerManager.kt

```diff
@@ -0,0 +1,15 @@
+package com.craftschemers.hub
+
+import org.bukkit.GameMode
+import org.bukkit.entity.Player
+
+class HubPlayerManager {
+    private var players = mutableMapOf<String, HubPlayer>()
+
+    fun addPlayer(player: Player) {
+        players[player.name] = HubPlayer(player)
```

Comment:
The player's name should not be used as a key since that should change. The player's UUID is a better choice.

Reply:
This is something that is temporary and will only be used when they are on the server - it cannot be changed